neurodsp/tests/sim/test_periodic.py

```diff
@@ -88,6 +88,15 @@ def test_sim_variable_oscillation():
         sig4 = sim_variable_oscillation(None, FS, freqs, cycle='asine', rdsym=rdsyms[1:])
 
 
+def test_sim_damped_oscillation():
+
+    sig1 = sim_damped_oscillation(N_SECONDS, FS, FREQ1, .1)
+    sig2 = sim_damped_oscillation(N_SECONDS, FS, FREQ1, 50)
+
+    assert sig1.sum() < sig2.sum()
```

Comment:
It took me a second the understand what this is testing - I think it would be worth a comment, something like: "check that signal with more dampening has lesser overall amplitude"

Also - am I reading the figure in the PR wrong... I think I would expect the sig2, with much higher damping (blue) to have a lesser sum than the lightly damped (yellow) one? Maybe I don't understand this test.